fixtures/parallelizer/remote.py

```diff
@@ -213,8 +213,8 @@ def _init_config(slave_options, slave_args):
     appliance_data = conf.slave_config.get("appliance_data", {})
     if ip_address in appliance_data:
         template_name, provider_name = appliance_data[ip_address]
-        conf.runtime["cfme_data"]["basic_info"]["appliance_template"] = template_name
-        conf.runtime["cfme_data"]["basic_info"]["appliances_provider"] = provider_name
+        conf.cfme_data["basic_info"]["appliance_template"] = template_name
+        conf.cfme_data["basic_info"]["appliances_provider"] = provider_name
```

Comment:
when you write to runtime, everything other is forgotten (like vddk_url and so ...). I don't know whether to copy everything to runtime and then modify or what ... :cry: